### PR TITLE
DO NOT MERGE: preparing content for categories, languages, time zones, and rules.

### PR DESCRIPTION
**Pull request by angelgolfer-ms**

### CODE REVIEW

File: api-reference/beta/api/mailfolder_list_messagerules.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+} -->
+```http
+HTTP/1.1 200 OK
+Content-type: application/json
+
+{
+  "@odata.context":"https://graph.microsoft.com/beta/$metadata#Me/mailFolders('inbox')/messageRules",
+  "value":[
+    {
+      "@odata.id":"https://graph.microsoft.com/beta/users('94447c6e-ea4c-494c-a9ed-d905e366c5cb@007e925e-fb5a-4f60-9cb6-58a21e692480')/mailFolders('inbox')/messageRules('AQAAAJ5dZp8=')",
```
Comment (nimeesh-msft) [inline api-reference/beta/api/mailfolder_list_messagerules.md]: Can we remove odata.id from the response for Message rule?  This is applicable to all places where odata.id is referred in MessageRule.

File: api-reference/beta/resources/location.md [hunk 2]
```diff
@@ -2,7 +2,28 @@
...
+
+Depending on how an event is created, expect Outlook to set the read-only 
+**locationType**, **uniqueId**, and **uniqueIdType** properties of the location as described in the following table. 
+
+| How event was created  | Property   | Expected value |
+|:----------|:-------|:--------------------------------|
+| [create event](../api/user_post_events.md) REST API | | |
+|  | **locationType** | `default` |
+|  | **uniqueId** | Same as the **displayName** property. |
+|  | **uniqueIdType** | `unknown` or `private` |
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/location.md]: Elias, I have seen "unknown" and "private" returned in a couple of events I programmatically created, but am not sure if the other values ("locationStore", "directory" and "bing") would possibly be returned for such an event, and so I haven't included them here. Please confirm.

Comment (EliasKaplan) [reply api-reference/beta/resources/location.md]: uniqueId
Same as the displayName or locationUri property. is not always, the UniqueId will be set to guid by internal processor.

File: api-reference/beta/resources/location.md [hunk 3]
```diff
@@ -2,7 +2,28 @@
...
+| User interface in Outlook | | |
+|  | **locationType** | One of the following: <ul><li>`default` for a location entered as plain text.</li><li>`conferenceRoom` for a room provided by the Outlook rooms list.</li><li>Or, any of this list - `homeAddress`, `businessAddress`,`geoCoordinates`, `streetAddress`, `hotel`, `restaurant`, `localBusiness`, `postalAddress` - for a location from Bing Autosuggest or Bing local search.</li></ul> |
+|  | **uniqueId** | Same as the **displayName** or **locationUri** property. |
```
Comment (EliasKaplan) [inline api-reference/beta/resources/location.md]: Is not always one of these values, the UniqueId will be set to guid by internal processor. better switch to some more gral like in the description of IcalUid:
A unique identifier that is shared by all instances of a location across events.

**Pull request merged**